src/gtfs/read.rs

```diff
@@ -1183,23 +1172,44 @@ where
     collections.routes = CollectionWithId::new(routes)?;
 
     gtfs_routes_collection.iter().for_each(|(_id, gtfs_route)| {
-        if let Some(comment) = generate_route_comment(&gtfs_route, read_as_line) {
-            if read_as_line {
-                if let Some(mut line) = collections.lines.get_mut(&gtfs_route.id) {
-                    line.comment_links.insert(comment.id.to_string());
-                    collections
-                        .comments
+        fn insert_comment<'c, T: typed_index_collection::Id<T> + objects::CommentLinks>(
```

Comment:
this function inside the for_each disrupts my reading. Can you put it outside please?

Reply:
I intentionally put it inside since it is of no use anywhere else. Anyway, it was already the case with the previous `generate_route_comment()` function, so I'll put it back there to replace it.

Done in 2254c72fb0024eff1cf1d95a04896866c94ac900.